src/bin/cron_calc.py

```diff
@@ -88,13 +85,29 @@ def main():
         assert sys.argv[1].lower() in tasks, f"sys.argv[1] must be: {'or '.join(tasks)}"
         tasks = sys.argv[1].lower()
 
+    # --------------------------------------
+    # Getting B3 and Binance Tickers
+    # --------------------------------------
+    TICKERS_IBRX = asyncio.run(load_ibrx100())
```

Comment:
TICKERS_IBRX and BINANCE_FUTURES are in fact global.
Many modules use this variables.

The best strategy is to read them from a config file